### ISSUE TITLE
Show humidity in temp tiles

### ISSUE THREAD

**Issue by The2Earthlings** [kwindrem/GuiMods#43]
It would  be great to display the humidity just after the temperature  if space allows it and ruuvi is able to.

Example: 13.2°C • 53%

**Comment by kittyorianda** [comment]
Would be happy if the integration of the humidity indicator from Ruuvi sensors would be possible.

**Comment by kwindrem** [comment]
There simply is not enough space for humidity, at least on the "compact" views where name and values need to fit on one line.

**Comment by kittyorianda** [comment]
With long names the system scrolls from right to left. Doesn't that work there too?

**Comment by kwindrem** [comment]
Yes, but if humidity were displayed there's be about 1 or 2 character spaces for the name to scroll in

**Comment by The2Earthlings** [comment]
Hi @kwindrem - in the view there should be enough space: first line "name of the sensor", second line "temperature and humidity, if not enough space in small views you can scroll or hide humidity.

Here an example how it can work

<img width="514" alt="view_temp_and_humidity" src="https://user-images.githubusercontent.com/100387182/169703201-96097178-200b-409f-b3d8-7f5d46df6de6.png">
k

**Comment by kwindrem** [comment]
Yes. It's the "compact" version where this won't work:
![Screen Shot 2022-05-22 at  8 31 16 AM](https://user-images.githubusercontent.com/58538395/169703305-ae09aa0e-4827-41f2-9c98-86cabf6e6913.png)

I can probably also make humidity fit in the Mobile overview even in the compact layout there:

![Screen Shot 2022-05-22 at  8 33 35 AM](https://user-images.githubusercontent.com/58538395/169703408-6d409aa0-3db4-4560-849e-f8343536ff76.png)

But names will need to be fairly short or scroll

**Comment by kittyorianda** [comment]
![image](https://user-images.githubusercontent.com/106024534/169703533-eeea47bf-2bd0-4028-b517-0a37a30a4839.png)

Thanks @ mine would also have space, even if I plan to add an additional 3 sensors.

Yes, if you have these views, then things get tight. But maybe you can make it selectable in the options, whether you want to have the simple version, or the detailed?

**Comment by kittyorianda** [comment]
A clickables field similar to touching the MPPT or AC loads boxes - etc., I read somewhere would probably not go?

**Comment by kwindrem** [comment]
The selection between the two-line and single line format is done automatically based on how many tanks and temps need to be shown. I can't make this a manual selection. Your only choice would be to hide tanks, then the temps would be two line in the flow view. In the mobile overview, it would still switch to single line mode if there were lots of temps.

Touch on the tanks and temps tile won't work because these are set up in a scrolling list so if there are lots of them, you can see them all by scrolling. Can't do both tap and drag to scroll.

**Comment by kittyorianda** [comment]
On the mobile view an additional third line and make the rest of the display above a little smaller?
